server/data/elite2ClientBuilder.js

```diff
@@ -49,5 +49,16 @@ module.exports = token => {
             : logger.warn(error, `Error calling elite2`),
       })
     },
+    getActiveAgenciesByType(type) {
```

Comment:
I added a `getPrisons` method to this class which you can re-use instead of this. 

I aimed for encapsulating the quirks of elite2 into just the client - no need to spread the knowledge that elite stores prisons as "agencies" of a type of `INST` anywhere but in here

Reply:
Done.
swapped over to use getPrisons